stable/storageos-operator/README.md

```diff
@@ -93,33 +123,29 @@ metadata:
   namespace: "default"
 spec:
   secretRefName: "storageos-api"
-  secretRefNamespace: "default"
+  secretRefNamespace: "storageos-operator"
+  csi:
+    enable: true
+    deploymentStrategy: "deployment"
+    enableProvisionCreds: true
+    enableControllerPublishCreds: true
+    enableNodePublishCreds: true
+    enableControllerExpandCreds: true
+  kvBackend:
+    address: "etcd-client.etcd.svc.cluster.local:2379"
+    # address: '10.42.15.23:2379,10.42.12.22:2379,10.42.13.16:2379' # You can set ETCD server IPs.
 ```
 
 Once the `StorageOSCluster` configuration is applied, the StorageOSCluster
-operator will create a StorageOS cluster in the `storageos` namespace by
+operator will create a StorageOS cluster in the `kube-system` namespace by
 default.
```

Comment:
Let's add a short reason why, so users aren't tempted to override:
> ... will create a StorageOS cluster in the `kube-system` namespace by
default.	Installing into the `kube-system` namespace is recommended so that the storage containers can be marked as `system-node-critical`.  This reduces the risk that the storage containers get evicted before applications that require the storage they provide.